bazel/python_utils/conftest.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+
+
+@pytest.hookimpl()
+def pytest_sessionstart(session):
+    if os.geteuid() != 0:
+        raise Exception(
+            "\n\n" + \
+            "################################################################################\n" * 3 + \
+            "To execute tests tagged as 'manual' you need to use the relevant shell script in\n" + \
+            "'$MAGMA_ROOT/bazel/scripts/'!\n" + \
+            "For Python sudo tests: '$MAGMA_ROOT/bazel/scripts/run_sudo_tests.sh'!\n" + \
+            "For LTE integration tests: '$MAGMA_ROOT/bazel/scripts/run_integ_tests.sh'!\n" + \
+            "For AGW load tests: '$MAGMA_ROOT/bazel/scripts/run_load_tests.sh'!\n" + \
```

Comment:
We can remove this line since the load tests do not require sudo. The load test is simply there to keep the order of the verbs as intended for the test cases.